packages/markedparser-extension/src/index.ts

```diff
@@ -148,8 +148,6 @@ namespace Private {
       async: true,
       // enable all built-in GitHub-flavored Markdown opinions
       gfm: true,
-      // santizing is applied by the sanitizer
-      sanitize: false,
```

Comment:
For posterity that option has been removed in Marked 8.0.0: https://github.com/markedjs/marked/pull/2936